ingest/defaults/config.yaml

```diff
@@ -7,6 +7,10 @@
 # Required to fetch from NCBI Datasets
 ncbi_taxon_id: "2560602"
 
+# Required to fetch from Entrez
+# Pull and spike in the `strain` field from Entrez
+entrez_search_term: "txid2560602[Primary Organism]"
+
```

Comment:
This config param can be removed since 792bda289fb1df9aceca2a251d5a0e28278614aa switches to constructing the search term.